orchestra/__init__.py

```diff
@@ -1,4 +1,4 @@
 # The current Orchestra version.
-__version__ = '1.0.49'
+__version__ = '1.0.50'
```

Comment:
Thank you for being proactive :). We don't bump versions this way --- we have a script (https://github.com/b12io/orchestra/blob/main/release.py) that will handle this once the code is on `main`.